etc/testing/auth/dex.yaml

```diff
@@ -18,18 +18,27 @@ config:
     tlsKey: /etc/dex/tls/https/server/tls.key
     allowedOrigins: []
   staticClients:
+  - id: testapp
+    name: "Test App"
+    redirectURIs:
+    - 'http://test.example.com:657/authorization-code/callback'
+    secret: test
```

Comment:
Add the new app that requests the token. IRL this would be Notebooks or something similar, and the user would do a normal auth code flow.